src/box/journal.h

```diff
@@ -128,6 +133,7 @@ journal_entry_create(struct journal_entry *entry, size_t n_rows,
 	entry->n_rows		= n_rows;
 	entry->res		= JOURNAL_ENTRY_ERR_UNKNOWN;
 	entry->flags		= 0;
+	entry->complete = false;
```

Comment:
Does the linter complain if you align this assignment by the other ones?

Reply:
Yes